test/integration/default/inspec/osquery_spec.rb

```diff
@@ -65,4 +66,27 @@
     it { should be_enabled }
     it { should be_running }
   end
+
+  describe json('/var/osquery/osquery.conf') do
+    expected_options = {
+      'syslog_pipe_path' => '/var/osquery/syslog_pipe',
+      'config_plugin' => 'filesystem',
+      'logger_plugin' => 'filesystem',
+      'schedule_splay_percent' => 10,
+      'events_expiry' => 3600,
+      'verbose' => false,
+      'worker_threads' => 2,
+      'logger_path' => '/var/log/osquery'
+    }
+
+    expected_schedule = {
+      'info' => { 'query' => 'SELECT * FROM osquery_info;', 'interval' => 86_400 }
```

Comment:
Same typo with `interval`

Reply:
Updated!

Reply:
TIL ruby can just add `_` anywhere in a number for readability, and rubocop demands it for thousands separation.